README.md

```diff
@@ -1,13 +1,120 @@
+```
+
+Great! Now let's use our mint to create 1000 new BaytownBucks.
+
+```js
+const purse = baytownBucksMint.mint(1000, 'community treasury');
+```
+
+The act of minting created 1000 BaytownBucks and stored them together in a
+`purse`. Purses in ERTP only hold one type of digital asset, so this
+purse can only ever hold BaytownBucks.
+
+Let's distribute the BaytownBucks to members of the community. To send
+money in ERTP, we withdraw `payments` from purses. 
+
+```js
+const paymentForAlice = purse.withdraw(10, `alice's community money`);
+```
+
+Like our purse, this payment contains BaytownBucks, but unlike purses,
+payments are used to represent tokens in transit. A payment can be
+sent to someone else, a purse never should be. 
+
+Now let's send the payment to Alice as message:
+
+```
+alice.receivePayment(paymentForAlice);
+```
+
+This may seem strange, but ERTP is built on top of [an
+infrastructure](https://github.com/Agoric/SwingSet) in which
+everything is an object. In this example, we have a reference to the
+object `alice`, and can call her `receivePayment` method to send her a
+payment. Alice's methods are entirely up to her, and are not part of
+ERTP.
+
+## Security Properties
+
+How does Alice know that she got paid real money? She could have been
+sent fake money, or she could have been sent money that was
+[double-spent](https://en.wikipedia.org/wiki/Double-spending). 
+
+When alice receives an alleged payment, she can call a method to know
+that the payment is 1) valid, and 2) exclusively hers:
+
+```js
+const myExclusivePayment = BaytownBucksIssuer.claimAll(allegedPayment);
+```
+
+The BaytownBucksIssuer is associated with the BaytownBucksMint, but
+the issuer is the public-facing version that is accessible to anyone.
+By holding the reference to a mint, you can mint more tokens. By
+holding a reference to the issuer for a mint, you can check that a
+payment is valid and exclusively claim it for yourself. 
+
+That's the basic use case for a fungible token. `makeMint` in
+[issuers.js](core/issuers.js) takes
+in an optional configuration that allows for many more possibilities. 
+
+## Pixel Gallery Demo
+
+To explore further types of digital assets, beyond fungible tokens, we
+decided to create a demo for our
+[testnet](https://github.com/Agoric/cosmic-swingset) in which users
+can buy and sell the right to color a pixel on a webpage. Read more
+[here](/GALLERY-README.md).
+
+## Gotchas
+
+When looking at the code in our [tests](test), you might see some new
+concepts:
+
+* __Vats__: All user code runs in what we call a `vat`. Within a vat,
+  code is run synchronously. Communication with another vat happens
+  asynchronously. The [SwingSet
+  infrastructure](https://github.com/Agoric/SwingSet) creates the vats
+  and makes communication between vats possible. 
+
+* __E() and infix bang (!)__: With our Swingset infrastructure, we can
+  talk to objects in another vat as if they were local, which is
```

Comment:
as if they were local? that's over-selling, no? The difference isn't just syntax; infix-bang sends are eventual; i.e. they return promises.

https://en.wikipedia.org/wiki/Fallacies_of_distributed_computing

Reply:
Perhaps introduce the syntax first with local objects, just as a way of invoking them asynchronously. Then, with that said, you can validly say that, using `E` or `!`, you can talk asynchronously to local and remote objects the same way, which is really cool!

Reply:
Sure, Mark, introducing the syntax with local objects first sounds like a great solution. Dan, I'm not seeing any of fallacies mentioned in the wiki page as applicable here.

Reply:
The text Dan was reacting to sounds like its saying that you can use "!" to talk to objects remotely just as you use "." to talk to objects locally. Indeed, ***almost all historic RPC systems*** tried to do exactly that, blocking the sender at that point waiting for a round trip to complete, in order to emulate a synchronous function call. Given that network latencies are huge compared to local procedure calls, this was a terrible idea. The original Fallacies paper was written in response to such architectures. "Latency is zero".

By introducing "!" first as something different from ".", it is then true that remote "!" is like local "!" because the we designed the semantics of "!" (and indeed of non-blocking promises) so that it stayed within the restrictions of what could be transparently stretched over the network.

Reply:
Ah, gotcha. I think for modern JavaScript programmers, they will not necessarily assume that local objects are accessed synchronously.

Reply:
Only once they understand that "!" is not just like ".", which the original text suggested.

Reply:
Sure, that could be made clearer